doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,141 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit image to function as a bluetooth speaker
+===========================================================================
+
+For configuring the bluetooth speaker setup you should use the
+"gateway" image and prepare couple of configuration files. This
```

Comment:
its part of refkit configuration. for example docker/local-build.sh -c build refkit-image-gateway